product_docs/docs/pem/9/tuning_performance/performance_diagnostic.mdx

```diff
@@ -14,7 +15,7 @@ The Performance Diagnostic dashboard analyzes the database performance for Postg
 To analyze the wait states data on multiple levels, narrow down the data you select. The data you select at the higher level of the graph populates the lower level.
 
 !!! Note
```

Comment:
Why do we mention RHEL and CentOS here? I think we can just remove this entire note.

Reply:
Slack thread for this note: https://edb.slack.com/archives/GDTA6A51P/p1698210038839939?thread_ts=1697796450.680159&cid=GDTA6A51P

Can we include this as note?
Performance Diagnostics is available for all the Postgres variants and versions, for which the EDB wait states module is available. EDB wait states is available on:
         • EDB Postgres Advanced Server and PostgreSQL version 11 and later
         • EDB Postgres Extended version 11 and later

Reply:
@sonotley, do you agree with this, or shall I remove the note?

Reply:
I would suggest simplifying it even further to the following, then we won't have to update it again when 11 goes EOL.

> Performance Diagnostics is available for all the Postgres variants and versions, for which the EDB wait states module is available.

Reply:
Done